src/amber/cli/templates/app/spec/spec_helper.cr

```diff
@@ -1,3 +1,5 @@
+ENV["AMBER_ENV"] ||= "test"
```

Comment:
Why is this set conditionally? User can accidentally export `AMBER_ENV`, forget about it and then run specs. ie:

```crystal
$ export AMBER_ENV=production
# do some stuff...
$ crystal spec
```

Reply:
The reason is that you can have the AMBER_ENV can be different for CI environment

Reply:
Would it be sufficient to exit when `AMBER_ENV == production`? Or perhaps issue a compile-time warning when `AMBER_ENV` already exists?

```crystal
{% if ENV["AMBER_ENV"]
puts "Warning: AMBER_ENV is set to #{ENV["AMBER_ENV"]}"
end %}
```

Reply:
@veelenga what I am trying to state is that having AMBER_ENV set to `test` will remove the ability from devs to use a different `environment.yml` config file for a continuous integration environment like travis.

See https://github.com/rspec/rspec-rails/blob/master/lib/generators/rspec/install/templates/spec/rails_helper.rb#L3

Reply:
I got. But look two lines below:

https://github.com/rspec/rspec-rails/blob/master/lib/generators/rspec/install/templates/spec/rails_helper.rb#L5-L6

You may do the same here.

Reply:
Nice catch!